### PR TITLE
Fix minibatch size

### PR DESCRIPTION
**Pull request by mbalesni**
The relevant change is only in `trlx/trainer/accelerate_base_trainer.py`. 

Other changes are irrelevant to this PR purpose but nice to have.

### CODE REVIEW

File: trlx/trainer/accelerate_base_trainer.py [hunk 1]
```diff
@@ -48,14 +48,15 @@ class AccelerateRLTrainer(BaseRLTrainer):
...
-        if config.train.minibatch_size:
-            assert config.train.batch_size % config.train.minibatch_size == 0, "Minibatch size must divide batch size"
-            self.mb_size = config.train.minibatch_size
+        self.accelerator = Accelerator(log_with=config.train.tracker, logging_dir=config.train.logging_dir)
+        if not config.train.minibatch_size:
+            config.train.minibatch_size = config.train.batch_size // self.accelerator.gradient_accumulation_steps
         else:
-            self.mb_size = config.train.batch_size
+            assert config.train.batch_size % config.train.minibatch_size == 0, "Minibatch size must divide batch size"
+            assert self.accelerator.gradient_accumulation_steps == 1, "Custom minibatch size is not compatible with grad accumulation"
```
Comment (meg-tong) [inline trlx/trainer/accelerate_base_trainer.py]: Don't we want to assert in the second case that:
```
config.train.batch_size == config.train.minibatch_size * self.accelerator.gradient_accumulation_steps
```
?

File: scripts/accelerate_launch_wrapper.py [hunk 2]
```diff
@@ -23,12 +25,16 @@
...
+
+        # arguments to accelerate.launch
         "--num_processes", str(args.num_processes),
         "--main_process_port", str((DEFAULT_MASTER_PORT + slurm_job_id + slurm_array_task_id) % 65000),
         "--gradient_accumulation_steps", str(args.gradient_accumulation_steps), 
         "--config_file", args.config_file,
         "--mixed_precision", args.mixed_precision,
-        args.experiment_file,
+
+        # this is an executable, next lines are arguments to it
```
Comment (meg-tong) [inline scripts/accelerate_launch_wrapper.py:36]: Ty for more comments here

File: trlx/trainer/accelerate_base_trainer.py (hunk 1 above)
Comment (mbalesni) [reply trlx/trainer/accelerate_base_trainer.py]: You're right! Fixed

**Review by meg-tong — APPROVED** [review]
Ty!

**Pull request merged**